test/e2e/preneed-helpers.js

```diff
@@ -0,0 +1,3004 @@
+    .fill('input[name="root_application_claimant_ssn"]', data.application.claimant.ssn)
+    .fillDate('root_application_claimant_dateOfBirth', data.application.claimant.dateOfBirth)
+    .selectRadio('root_application_claimant_relationshipToVet', data.application.claimant.relationshipToVet);
+
+  if (data.application.claimant.relationshipToVet.type === 'other') {
+    client
+      .waitForElementVisible('input[name="root_application_claimant_relationship_other"]', Timeouts.normal)
+      .fill('input[name="root_application_claimant_relationship_other"]', data.application.claimant.relationship.other)
+      // Not sure what to do with this, exactly, but I'll make it an option.
+      .clickIf('#root_application_claimant_relationship_view:isEntity', data.application.claimant.relationship.isEntity);
+  }
+}
+
+function completeVeteranInformation(client, data) {
+  client
+    .fillName('root_application_veteran_currentName', data.application.veteran.currentName)
+    .fill('input[name="root_application_veteran_ssn"]', data.application.veteran.ssn)
+    .fill('input[name="root_application_veteran_militaryServiceNumber"]', data.application.veteran.militaryServiceNumber)
+    .fillDate('root_application_veteran_dateOfBirth', data.application.veteran.dateOfBirth)
+    .fill('input[name="root_application_veteran_placeOfBirth"]', data.application.veteran.placeOfBirth)
+    .selectRadio('root_application_veteran_gender', data.application.veteran.gender)
+    .selectRadio('root_application_veteran_maritalStatus', data.application.veteran.maritalStatus)
+    .selectDropdown('root_application_veteran_militaryStatus', data.application.veteran.militaryStatus)
+    .selectRadio('root_application_veteran_isDeceased', data.application.veteran.isDeceased)
+    .fillDate('root_application_veteran_dateOfDeath', data.application.veteran.dateOfDeath);
+}
+
+function completeServicePeriods(client, data) {
+  data.serviceRecords.forEach((tour, index) => {
+    client
+      .fillDate(`root_application_veteran_serviceRecords_${index}_dateRange_from`, tour.dateRange.from)
+      .fillDate(`root_application_veteran_serviceRecords_${index}_dateRange_to`, tour.dateRange.to)
+      .selectDropdown(`root_application_veteran_serviceRecords_${index}_serviceBranch`, tour.serviceBranch)
+      .fill(`input[name="root_application_veteran_serviceRecords_${index}_highestRank"]`, tour.highestRank)
+      .selectDropdown(`root_application_veteran_serviceRecords_${index}_dischargeType`, tour.dischargeType);
+
+    // Keep adding them until we're finished.
+    if (index < data.serviceRecords.length - 1) {
+      client.click('.usa-button-secondary.va-growable-add-btn');
+    }
+  });
+}
+
+function completeServiceName(client, data) {
+  client.selectRadio('root_application_veteran_view:hasServiceName', 'Y')
+    .fillName('root_application_veteran_serviceName', data.serviceName);
+}
+
+function completeBenefitSelection(client, data) {
+  // TODO: Include autosuggest interaction
```

Comment:
Right now this test is not making use of the autosuggest functionality, it is simply filling the field. It seems like a more robust implementation that makes use of the getCemeteries mock request, and the autosuggest functionality that involves only partially filling the field and then selecting one of the suggested options should be built as well? Should a ticket be created for that or is that beyond the scope of the e2e tests?